landing-page/content/services/hidden-partitioning.html

```diff
@@ -1,6 +1,8 @@
 ---
 Title: Hidden Partitioning
-Img: partition-spec-evolution.png
+Description: Iceberg handles the tedious and error-prone task of producing partition values for rows in a table and avoids reading unnecessary partitions automatically. Consumers don’t need to know how the table is partitioned or add extra filters to their queries and the partition layouts can evolve as needed.
```

Comment:
"Avoids reading" can be replaced with "skips unnecessary partitions and files automatically"

Reply:
I'd remove "consumers" because I don't think it is clear who a "consumer" is. Instead, I'd use a more direct phrase, like "No extra filters are needed for fast queries, and table layout can be updated as data or queries change."

Reply:
Fixed! Relevant commit [fea5ec9](https://github.com/apache/iceberg-docs/pull/18/commits/fea5ec9862883a725a7b7d0e5385281d975b8c15)